### ISSUE TITLE
Just unblock me

### ISSUE THREAD

**Issue by RedstoneGunMade** [RandomAltThing/PocketPC-Addon#1]
if you want, we can rename the project to PocketPC

**Comment by RandomAltThing** [comment]
I've already done alot of stuff but k

**Comment by RandomAltThing** [comment]
I also added the bool duelWield and when false it doesn't display 2 hands (Dodsn render any hands it's a bug

**Comment by RandomAltThing** [comment]
But when set to true it renders 2 hands correctly.

**Comment by RandomAltThing** [comment]
I'm not in the org I can't push it

**Comment by RedstoneGunMade** [comment]
ok
